examples/src/client/tracing_datasets.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-process-env */
 import { ChatOpenAI } from "langchain/chat_models/openai";
-import { LangChainPlusClient, Dataset } from "langchain/client";
+import { runOnDataset } from "langchain/client";
+import { LangChainPlusClient, Dataset } from "langchainplus-sdk";
```

Comment:
Importing from `"langchain/client"` and then not importing `LangChainPlusClient` from that entrypoint looks a little odd to me

Reply:
Ya I'm not sure what to call the directory now. Could be langchainplus but then will likely rename anyway

Reply:
It feels like `runOnDataset` specifically seems like it should be in `langchainplus-sdk` - `Dataset` is imported from the SDK and it's odd that the concept of a dataset wouldn't be self-contained. There's no way to abstract out the LangChain specific references from this method and just put everything in the new SDK then?

Reply:
Could we frame this as a "LangChain LangChain Plus adapter" maybe?

Reply:
Hm there's a lot of types to include: llms, chat models, callbacks. I think that could be in another PR.

While it would be nice to have run utils in the SDK, they're only relevant to langchain primitives so I'm not sure we'd need to bundle them there